test/functional/NativeTest/playlist.xml

```diff
@@ -569,4 +570,31 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		</subsets>
 		<disabled>https://github.com/eclipse/openj9/issues/3561</disabled>
 	</test>
+	<test>
+		<testCaseName>TestInvtest</testCaseName>
+		<command>cp -R $(TEST_JDK_HOME)/ $(REPORTDIR_NQ)/subdir; \
```

Comment:
Please do not use `subdir ` as a folder name. Name some meaningful. For example, testJava